crates/matrix-sdk/src/client/login_builder.rs

```diff
@@ -142,6 +142,7 @@ impl LoginBuilder {
     /// Instead of calling this function and `.await`ing its return value, you
     /// can also `.await` the `LoginBuilder` directly.
     #[instrument(
+        parent = &self.client.root_span,
```

Comment:
We should do the same thing for `SsoLoginBuilder`, I think.

Reply:
Yep, added in https://github.com/matrix-org/matrix-rust-sdk/pull/1447/commits/da28114792348d052c40aab75fa7821e2b2bf559.